iot_verifier/src/poc.rs

```diff
@@ -451,6 +453,7 @@ fn verify_beacon_payload(
         InvalidReason::InvalidPacket
     })?;
     tracing::debug!("generated beacon {:?}", generated_beacon);
+    println!("generated beacon {:?}", generated_beacon);
```

Comment:
Did you mean to leave these `println!` in?

Reply:
removed